backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -309,6 +312,7 @@ export class TransactionLinkResolver {
         throw new Error(`Creation from contribution link was not successful. ${e}`)
       } finally {
         await queryRunner.release()
+        releaseLock()
       }
       return true
```

Comment:
Is this return statement in the right place? In the transaction resolver it is inside the try block

Reply:
that doesn't matter in my opinion. It was unclear to me if a `finally` block is executed after a `return` statement. This is the case.
In this particular part of the code its the other case - the finally block is definitely executed, because the return comes after it.

My opinion is that this code (return after finally) is better for readability and therefore to be preferred over the other variant.
I would like to keep it as it is, since its no impact on how things are executed, but has the least code impact.